pynq/lib/intf/waveform.py

```diff
@@ -30,7 +30,7 @@
 import os
 import json
 import IPython.core.display
-from .intf_const import PYNQZ1_DIO_SPECIFICATION
+from .intf_const import BOARD_DIO_SPECIFICATION
```

Comment:
No - this is not how we will do this constant.  The Overlay defines the spec, not the board.